src/content/en/updates/2017/10/devtools-release-notes.md

```diff
@@ -0,0 +1,145 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Multi-client remote debugging, push notifications with custom data, and Workspaces 2.0.
+
+{# wf_updated_on: 2017-10-11 #}
+{# wf_published_on: 2017-10-11 #}
+{# wf_tags: chrome63,devtools,devtools-whatsnew #}
+{# wf_featured_image: /web/updates/images/generic/chrome-devtools.png #}
+{# wf_featured_snippet: Multi-client remote debugging, push notifications with custom data, and Workspaces 2.0. #}
+
+{% include "web/tools/chrome-devtools/_shared/styles.html" %}
+
+# What's New In DevTools (Chrome 63) {: .page-title }
+
+{% include "web/_shared/contributors/kaycebasques.html" %}
+
+Note: The video version of these release notes will be published around
+early-December 2017.
+
+Welcome back! New features coming to DevTools in Chrome 63 include:
+
+* [Multi-client remote debugging support](#multi-client).
+* [Workspaces 2.0](#workspaces).
+* [Simulate push notifications with custom data](#push).
+* [Trigger background sync events with custom tags](#sync).
+
+Note: You can check what version of Chrome you're running at
+`chrome://version`. Chrome auto-updates to a new major version about every 6
+weeks.
+
+## Multi-client remote debugging support {: #multi-client }
+
+If you've ever tried debugging an app from an IDE like VS Code or WebStorm,
+you've probably discovered that opening DevTools messes up your debug session.
+This issue also made it impossible to use DevTools to debug WebDriver tests.
+See the video below for an example of the issue in VS Code.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="OJlOU6UecFU"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+As of Chrome 63, this problem is no more. DevTools now supports multiple
+remote debugging clients. Watch the video below to see an example of VS Code
+and DevTools in action, side-by-side.
+
+<div class="video-wrapper-full-width">
+  <iframe class="devsite-embedded-youtube-video" data-video-id="tTWs0ZyXI-0"
+          data-autohide="1" data-showinfo="0" frameborder="0" allowfullscreen>
+  </iframe>
+</div>
+
+## Workspaces 2.0 {: #workspaces }
+
+Workspaces have been around for some time in DevTools. This feature enables
+you to use DevTools as your IDE. You make some changes to your source code
+within DevTools, and the changes persist to the local version of your
+project on your file system.
+
+Workspaces 2.0 builds off of 1.0, adding a more helpful UX and improved
```

Comment:
General comment that this section is a bit 'thisy'. Suggestion to remove some of the thises.